File: JimOrders.py
// https://www.hackerrank.com/challenges/jim-and-the-orders/problem

def jimOrders(orders):
    if len(orders) == 1:
        return [1]
    
    queue = []
    
    for i in range(len(orders)):
        queue.append([sum(orders[i]),i+1])

    queue.sort()

    return [i[1] for i in queue]
    
    

File: JimOrders.cpp
// https://www.hackerrank.com/challenges/jim-and-the-orders/problem

vector<int> jimOrders(vector<vector<int>> orders) {

    multimap<int,int> m;

    for (int i=0; i<orders.size(); i++)
    {
        m.insert(make_pair(orders[i][0]+orders[i][1], i+1));
    }   
    
    vector<int> result;
    
    for(auto me: m)
    {
        result.push_back(me.second);
    }   
    
    return result;
}
